index.html

```diff
@@ -114,7 +114,7 @@
    <section class="intro" id="home">
      <div class="main-content">
         <p class="quote">&#34;hello world!welcome to&#34;</p>
-        <h2 class="header-content">chat<span class="gpt">gpt</span> forum: fast coding.the future of software development</h2>
+        <h1 class="header-content">chatgpt annual global forum<br> 2023</h1>
         <p class="descr">A joyful celebration of the succesful launch of the ChatGPT(Chat Generative Pre-trained Transformer) model.Meet developers from over 105 countries during the celebration in San Franscisco,California </p> 
         <p class="date">2023.12.11(TUE)~(WED)</p>  
         <p class="address">&#169;AMA Conference Center San Francisco and more</p> 
```

Comment:
- [ ] `required`
- [ ] Amazing job so far 👍 
- [ ] however, there is a few things you to put in place.
- [ ] please follow the fonts size as close as possible
- [ ] also i suggest you add the border to the paragraph
- [ ] please see screenshots below

`your template`
![image](https://user-images.githubusercontent.com/66230306/218247906-89b927b2-7f30-4e9d-acf9-e8f3fbfaa1db.png)

`Cindy shin template`
![image](https://user-images.githubusercontent.com/66230306/218248084-7ed68204-557a-4134-9914-f6d1bc33c73e.png)